js/ui/multi_view.js

```diff
@@ -153,6 +153,7 @@ const MultiView = CollectionWidget.inherit({
         this._$itemContainer.appendTo(this._$wrapper);
 
         this.option('loopItemFocus', this.option('loop'));
+        this._findBoundaryAvailableIndices();
```

Comment:
Seems like meaningfulness of the name does not worsen if we change it to:
```suggestion
        this._findBoundaryIndices();
```
Am i wrong?

Reply:
Thank you! Was fixed in [1254ebd2f865395198a8f28647dccaf61766bada].